tcmu-runner.spec

```diff
@@ -0,0 +1,75 @@
+Name:          tcmu-runner
```

Comment:
I think you added this by mistake.

Reply:
Just to be clear, I mean I think you added that patch or file by mistake, because we already added a spec.